modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdRequestObjectTestCase.java

```diff
@@ -344,80 +363,46 @@ public void testClaims() throws Exception {
         BufferedReader rd = new BufferedReader(new InputStreamReader(response.getEntity().getContent()));
 
         Object obj = JSONValue.parse(rd);
-        String email = ((org.json.simple.JSONObject) obj).get("email").toString();
-        String givenName = ((org.json.simple.JSONObject) obj).get("given_name").toString();
+        String email = ((JSONObject) obj).get("email").toString();
+        String givenName = ((JSONObject) obj).get("given_name").toString();
 
         EntityUtils.consume(response.getEntity());
         Assert.assertEquals(USER_EMAIL, email, "Incorrect email claim value");
         Assert.assertEquals(GIVEN_NAME, givenName, "Incorrect given_name claim value");
         // externalClaim1 and externalClaim2 are not included in the requested scopes.
-        Assert.assertNull(((org.json.simple.JSONObject) obj).get("externalClaim1"));
-        Assert.assertNull(((org.json.simple.JSONObject) obj).get("externalClaim2"));
+        Assert.assertNull(((JSONObject) obj).get("externalClaim1"));
+        Assert.assertNull(((JSONObject) obj).get("externalClaim2"));
     }
 
     @Test(groups = "wso2.is", description = "Validate Token Expiration Time",
             dependsOnMethods = "testValidateAccessToken")
     public void testValidateTokenExpirationTime() throws Exception {
-        OAuth2TokenValidationRequestDTO requestDTO = new OAuth2TokenValidationRequestDTO();
```

Comment:
Shall we add the missing new line. Let's check in other places as well.

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096